quickstart.md

```diff
@@ -57,24 +38,16 @@
     }
     ```
 
-    The curl command issued an API request to analyze the `comment.text` field
-    for the `requestedAttributes`, in this case the `TOXICITY` model.
+    The curl command issued an API request to analyze the `comment.text` field for the `requestedAttributes`, in this case the `TOXICITY` model.
 
-    In the response ,the field `attributeScores.TOXICITY.summaryScore.value`
-    gives the toxicity model's score for the comment. In this case, the comment
-    got a 0.9 out of 1.0. A less mean-spirited comment should get a lower score.
+    In the response ,the field `attributeScores.TOXICITY.summaryScore.value` gives the toxicity model's score for the comment. In this case, the comment got a 0.9 out of 1.0. A less mean-spirited comment should get a lower score.
     
 Users can leverage the ‘DoNotStore’ flag to ensure that all submitted comments are automatically deleted after scores are returned and/or the ‘SuggestCommentScore’ method to submit corrections to improve Perspective over time.  
 
 See the [API reference documentation](api_reference.md) for details on all of
 the request and response fields, as well as the available values for
-`requestedAttributes`. There are quite a few
-[experimental models](https://github.com/conversationai/perspectiveapi/blob/master/api_reference.md#models),
-such as "obscene", "attack on a commenter", "spam", etc, that you may find
-interesting.
+`requestedAttributes`. There are quite a few [experimental models](https://github.com/conversationai/perspectiveapi/blob/master/api_reference.md#models), such as "obscene", "attack on a commenter", "spam", etc, that you may find interesting.
 
-Also, subscribe to our
-[perspectiveapi-announce@ Google Group](https://groups.google.com/forum/#!forum/perspective-announce/join) to
-get notified when we make changes to the API.
+Also, subscribe to our [perspectiveapi-announce@ Google Group](https://groups.google.com/forum/#!forum/perspective-announce/join) to get notified when we make changes to the API.
 
-For support and to contact us, see: https://support.perspectiveapi.com/. 
+For support and to contact us, see: support.perspectiveapi.com. 
```

Comment:
Also lets mention the quota limit of 1 qps, with a link to where they can request an increase.  Thanks!